Userland/Applications/Screenshot/ScreenshotWidget.cpp

```diff
@@ -113,9 +113,9 @@ void ScreenshotWidget::save_screenshot(String output_path, bool selected, bool c
         return;
     }
 
-    if (selected) {
-        bitmap = bitmap->cropped(crop_region);
-    }
+    // if (selected) {
+    //     bitmap = bitmap->cropped(crop_region);
+    // }
```

Comment:
You should remove commented out code from the commits.

Reply:
Oops! Thought I removed that.